### DISCUSSION TITLE
Trying to install telsamate on Ubuntu server 20.10 and getting next error

### DISCUSSION THREAD

**Discussion by slavag** [teslamate-org/teslamate#1499] [Q&A]
Trying to install telsamate on Ubuntu server 20.10 and getting next error :

** (CompileError) config/config.exs:23: undefined function config_env/0
    (stdlib 3.13) lists.erl:1354: :lists.mapfoldl/3
    (elixir 1.10.3) expanding macro: Kernel.to_string/1
    config/config.exs:23: (file)
    (elixir 1.10.3) expanding macro: Config.import_config/1
    config/config.exs:23: (file)
** (CompileError) config/config.exs:23: undefined function config_env/0
    (stdlib 3.13) lists.erl:1354: :lists.mapfoldl/3
    (elixir 1.10.3) expanding macro: Kernel.to_string/1
    config/config.exs:23: (file)
    (elixir 1.10.3) expanding macro: Config.import_config/1
    config/config.exs:23: (file)

Please advise.
Thanks

**Comment by adriankumpf** [comment]
Take a look at the installation steps provided here. You need Elixir v1.11 https://docs.teslamate.org/docs/installation/debian
